{{cookiecutter.project_name}}/setup.cfg

```diff
@@ -6,6 +6,12 @@ description-file =
 author = {{ cookiecutter.author_name }}
 author-email = {{ cookiecutter.author_email }}
 home-page = http://www.sphinx-doc.org/
+build_sphinx=
```

Comment:
I don't think this is a valid option for the `[metadata]` section. Rather, it should be in its own `[build_sphinx]` section.